integration-tests/src/test/java/oracle/weblogic/kubernetes/ItIntrospectVersion.java

```diff
@@ -151,6 +162,11 @@
 
   private Map<String, OffsetDateTime> podsWithTimeStamps = null;
 
+  private static final String INTROSPECT_DOMAIN_SCRIPT = "introspectDomain.sh";
+  private static final Path samplePath = Paths.get(ITTESTS_DIR, "../kubernetes/samples");
+  private static final Path tempSamplePath = Paths.get(WORK_DIR, "sample-testing");
```

Comment:
Can you use some unique path here? I am not sure if this same path was used in some other sample tests. It will pose a problem when running in parallel.

Reply:
changed